CODEOWNERS

```diff
@@ -1,88 +1,250 @@
+
 /specification/consumption/ @kjeur @panda-wang
+
+# PRLabel: %Container Instances
 /specification/containerinstance/ @novinc
+
+# PRLabel: %Container Registry
 /specification/containerregistry/ @djyou
+
+# PRLabel: %Container Service
 /specification/containerservice/ @palma21 @weinong @seguler @alvinli222 @justindavies @matthchr @robbiezhang @paulgmiller @yizhang4321 @circy9 @qike-ms
+
+# PRLabel: %Cosmos
 /specification/cosmos-db/ @dmakwana @MehaKaushik @wmengmsft @shurd
+
+# PRLabel: %Customer Insights
 /specification/customer-insights/ @tjlvtao
+
+# PRLabel: %Data Factory
 /specification/datafactory/ @Frey-Wang @zhangyd2015 @davidzhaoyue
+
+# PRLabel: %Data Lake Analytics
 /specification/datalake-analytics/ @ro-joowan
+
+# PRLabel: %Data Lake Store
 /specification/datalake-store/ @ro-joowan
+
+# PRLabel: %Data Migration
 /specification/datamigration/ @hitenjava
+
 /specification/deploymentmanager/ @netrock
+
+# PRLabel: %Devtestlab
 /specification/devtestlabs/ @iouri-s
+
+# PRLabel: %Network - DNS
 /specification/dns/ @owolabi2
+
 /specification/documentdb/ @dmakwana
+
 /specification/domainservices/ @jihochang
+
+# PRLabel: %Event Grid
 /specification/eventgrid/ @Kishp01 @ahamad-MS
+
+# PRLabel: %Event Hubs
 /specification/eventhub/ @v-ajnava @dsouzaarun @damodaravadhani
+
 /specification/features/ @stankovski
+
 /specification/graphrbac/ @lmazuel @yugangw-msft @amarzavery
+
+# PRLabel: %HDInsight
 /specification/hdinsight/ @pulkittomar @wawon-msft
+
 /specification/insights/ @gucalder
+
+# PRLabel: %Intune
 /specification/intune/ @vrmurthy01
+
+# PRLabel: %IotHub
 /specification/iothub/ @rkmanda
+
+# PRLabel: %KeyVault
 /specification/keyvault/ @heaths @randallilama @jlichwa
+
+# PRLabel: %Logic App
 /specification/logic/ @pankajsn @tonytang-microsoft-com
+
+# PRLabel: %Machine Learning
 /specification/machinelearning/ @nonstatic2014
+
+# PRLabel: %Machine Learning Compute
 /specification/machinelearningcompute/ @shutchings
+
 /specification/managementpartner/ @jeffrey-ace
+
+# PRLabel: %Media Services
 /specification/mediaservices/ @giakas
+
+# PRLabel: %Mobile Engagement
 /specification/mobileengagement/ @pshrosbree
+
+# PRLabel: %Monitor
 /specification/monitor/ @gucalder
+
+# PRLabel: %MySQL
 /specification/mysql/ @qingqingyuan
+
+# PRLabel: %Network
 /specification/network/ @MikhailTryakhov
+
+# PRLabel: %Notification Hub
 /specification/notificationhubs/ @amolr @smithab
+
+# PRLabel: %Monitor - Operational Insights
 /specification/operationalinsights/ @sw47
+
+# PRLabel: %Operations Management
 /specification/operationsmanagement/ @dashimi16
+
+# PRLabel: %Peering
 /specification/peering/ @rileymckenna
+
+# PRLabel: %Policy Insights
 /specification/policyinsights/ @pilor
+
+# PRLabel: %PostgreSQL
 /specification/postgresql/ @qingqingyuan
+
 /specification/powerbidedicated/ @tarostok
+
+# PRLabel: %PostgreSQL
 /specification/provisioningservices/ @kvish
+
```

Comment:
```suggestion

# PRLabel: %Purview
/specification/purview/ @mikekistler

```

Reply:
Should it be `/specification/purview/data-plane/` ?